pkgs/development/python-modules/construct/2.10.54.nix

```diff
@@ -14,7 +14,7 @@ buildPythonPackage rec {
     sha256 = "1mqspsn6bf3ibvih1zna2glkg8iw7vy5zg9gzg0d1m8zcndk2c48";
   };
 
-  checkInputs = [ pytestCheckHook pytest-benchmark enum34 numpy ];
+  checkInputs = [ pytestCheckHook enum34 numpy ];
```

Comment:
`enum34` can be removed as well. The module only supports [Python > 3.6](https://github.com/construct/construct/blob/master/setup.py#L19).

Could you please add `disabled = pythonOlder "3.6";` to make sure that we respect upstream's limit of the Python release?

Reply:
Actually, I just checked and 2.10.54 is only there for Python 2 support (search for `2.10.54` in `all-packages.nix`).

Reply:
You are right.